browser/src/control/Control.NotebookbarBuilder.js

```diff
@@ -1065,11 +1065,12 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 
 			var hasManyChildren = childData.children && childData.children.length > 1;
 			if (hasManyChildren) {
-				var tableId = childData.id ? childData.id.replace(' ', '') : '';
+				var tableId = childData.id ? childData.id.replace(' ', '') + '-container': String(parseInt(Math.random() * 1000));
```

Comment:
there is function `_makeIdUnique` in parent class (JSDialogBuilder) for that

Reply:
Thanks for checking. I left this part as is for now. It doesn't assign an id if table doesn't have one.